demos/food-and-beverage/simple-supplier-distributor/README.md

```diff
@@ -184,18 +184,13 @@ You can add data using the GraphQL Explorer view from either node in the Uni.
 1. Add a `Product` item using the mutation below
    ```graphql
    mutation AddProduct {
-     add_Product_async(
-       input: {category: natural, description: "Organic", name: "Orange Juice", price: 3.99, sku: "00008", supplier: "Wild Harvest"}
-     ) {
-       error
-       result {
-         _id
-         _owner
-         submissionTime
-         transactionId
-         version
-       }
-     }
+     add_Product(
+      input: {category: natural, description: "Organic", name: "Orange Juice", price: 3.99, sku: "00008", supplier: "Wild Harvest"}, syncMode: ASYNC) {
+        transaction {
+            _id
```

Comment:
```suggestion
```

`_id` won't be available given `ASYNC` mode, correct?  The reference to `_id` is repeated throughout the updates.  Maybe I don't fully understand the new interface.  I believe `_id` is only available in Sync (not Async) modes, and the `_id` will be the Vendia `_id` of the created entity.  

Did I misunderstand or is `_id` not expected here in the Async case?

Reply:
`_id` is available in ASYNC mode:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/95232537/158831209-85a29446-7e5b-4104-8b70-4e5dd9d7315c.png">

Reply:
I'm not sure why it has a different value than `transactionId`, though...

Reply:
From Ryan Green: "`_id` is the id of the object, same as `result._id`. I don’t love that but it’s a leftover from previous mutations, a bit hard to change now"

I think I'll update all these examples to use `transactionId`. I assume that's what customers would actually want.